### PR TITLE
[QA] Inbox / Active Bids spacing

### PR DESCRIPTION
**Pull request by mennenia**
Fixes https://github.com/artsy/collector-experience/issues/776

## Bids
simulator screen shot - iphone 6 - 9 1 - 2017-12-07 at 14 19 38

## Messages
simulator screen shot - iphone 6 - 9 1 - 2017-12-07 at 15 03 00

## iPad

simulator screen shot - ipad pro 12 9 inch - 2017-12-07 at 15 04 33

Skip New Tests

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/Inbox/ActiveBids/__tests__/index-tests.tsx`
- `src/lib/Components/Inbox/Conversations/__tests__/index-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by damassi** [comment]
@mennenia - just a heads up about https://github.com/artsy/emission/pull/874 fixing https://github.com/artsy/collector-experience/issues/787!

**Comment by l2succes** [comment]
After taking a closer look at the spec


Either spacing or the font size looks off on the Bid status label in the Active Bid row

File: src/lib/Components/Inbox/ActiveBids/index.tsx [hunk 1]
```diff
@@ -6,7 +6,14 @@ import { View } from "react-native"
 import { LargeHeadline } from "../Typography"
 import ActiveBid from "./ActiveBid"
 
-const Container = styled.View`margin: 20px 0 40px;`
+const Container = styled.View`
+  margin-top: 20px;
```
[the remark this answers is not in the document]
Comment (mennenia) [reply src/lib/Components/Inbox/ActiveBids/index.tsx]: I know, I just like to write it out. I personally find it easier to understand and to know what to change :)

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/Inbox/ActiveBids/__tests__/index-tests.tsx`
- `src/lib/Components/Inbox/Conversations/__tests__/index-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by mennenia** [comment]
Yeah, on the spec the letters look slightly skinnier, and I'm not sure if that has to do with the sketch mockup. I can't really do much because Avant Garde is just rendered a little chunky. You can see that the style is the same as the partner names (apart from the colours 😄), with the same font size (11px off top of my head). 

@katarinabatina mentioned to basically move the label to its new positioning, and I left any changes to the text style the same. 

Whatcha think?

**Comment by l2succes** [comment]
Yeah, that totally makes sense. I get that the font is rendered differently than it does in Sketch that part we can't do much about, that's fine. What makes it look different to me is the space between the first and second line. It seems very tight in comparison to the spec but if you think it's fine, let's leave it as is.

**Comment by mennenia** [comment]
Nice one 👍 

Added some margin, now looks like:

screen shot 2017-12-08 at 09 41 10

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/Inbox/ActiveBids/__tests__/index-tests.tsx`
- `src/lib/Components/Inbox/Conversations/__tests__/index-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Pull request merged**